docs/_pages/numerictypes.md

```diff
@@ -40,7 +40,7 @@ Notice that `Should().Be()` and `Should().NotBe()` are not available for floats
 
 ```csharp
 float value = 3.1415927F;
-value.Should().BeApproximately(3.14F, 0.01F);
+value.Should().BeApproximately(3.14F, 0.001F);
```

Comment:
I actually intended to say that the value must be between `3.13F` and `3.15F`, so unless my brain is somehow short circuiting, the original example is fine.

Reply:
I also don't get the point :)

It is also between 2 and 4
...and 1 and 5
...and -100 and 100